akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -463,7 +463,10 @@ final class ORSet[A] private[akka] (
         elementsMap
     }
     clearAncestor()
-    val newVvector = vvector.merge(that.vvector)
+
+    val (thatVvectorNode, thatVvectorVersion) = thatDot.versionsIterator.toList.head
+    val thatVvector = VersionVector(thatVvectorNode, thatVvectorVersion)
```

Comment:
isn't `thatVvector` equal to `thatDot`?
```
val newVvector = vvector.merge(thatDot)
```

Reply:
Absolutely, Patrik thanks for spotting this. I had here projection on `that.vvector`, later realised that this could be just `thatDot` but forgot to refactor fully.